lib/src/downloader.dart

```diff
@@ -90,6 +90,8 @@ class FlutterDownloader {
   /// external storage. If you want to save the file in the public Downloads
   /// directory instead, set [saveInPublicStorage] to true. In that case,
   /// [savedDir] will be ignored.
+  ///
+  /// [timeout] is added for android devices to manually change connection timeout.
```

Comment:
This section is already titled `Android-only` (see line 76). 

```suggestion
  /// [timeout] is the HTTP connection timeout.
```